integration_tests/utils.py

```diff
@@ -139,15 +139,15 @@ def cluster_fixture(
             ini = chain_data / cluster.SUPERVISOR_CONFIG_FILE
             ini.write_text(
                 re.sub(
-                    r"^command = (.*/)?chain-maind",
-                    "command = chain-maind-inst "
+                    r"^command = (.*/)?cronosd",
```

Comment:
We should remove this function, it's copied from chain-main, not used in cronos, there's `setup_cronos` function in the `network.py`.

Reply:
I found it is portable if I create multiple nodes with pytest-xdist. And it works in cronos as well.

Reply:
It is ok to use `setup_cronos`, I can try it to create multiple nodes.